sitback-furniture/src/components/header/header.jsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from "react";
+import { Link, useLocation, useParams } from "react-router-dom";
+import Button from "../button/button";
+import styles from './header.module.scss';
+import { headerData } from "../../constants/string-constant";
+
+const Header = ()=>
+{
+    const {categoryId} = useParams();
+    const location = useLocation();
+    const tabList = ['couches','chairs','dining'];
+    const [selectedTab,setSelectedTab] = useState();
+
+    function onButtonClick(buttonValue)
+    {
+        setSelectedTab(buttonValue);
+    }
+
+    useEffect(()=>{
+        setSelectedTab(categoryId);
+    },[location])
+
+    const navButtonSection = tabList.map((val,index)=>(
+        <Link key={index} to={`/shopping/${val}`}>
```

Comment:
Can we use index as key?

Reply:
it can be used but it will affect the performance